src/lib/components/forms/input/file-input/terra-file-input.component.scss

```diff
@@ -3,40 +3,5 @@
 .file-select {
     .file-preview {
         flex: 1;
-        width: 100%;
-        background-color: var(--color-structure-0);
-        box-shadow: var(--box-shadow);
-        padding: var(--space-lg);
-        display: flex;
-        border-radius: var(--border-radius);
-        margin-bottom: 2px;
-        min-height: 150px;
-        position: relative;
-        flex-direction: column;
-
-        > span {
-            width: 100%;
-            height: 100%;
-            display: flex;
-            background-size: contain;
-            background-repeat: no-repeat;
-            background-position: center center;
-            align-items: center;
-            flex: 1;
-
-            > [class^='icon-'] {
-                display: inline-flex;
-                margin: 0 auto;
-                font-size: 7.5em;
-
-                > [class^='path']:before {
-                    color: var(--color-structure-9);
-                }
-            }
-        }
```

Comment:
Can these lines really be removed?
The file-preview is still exisiting

Reply:
@adumitrescu-plenty

Reply:
This was not included in the first merge. 
<img width="1631" alt="Bildschirmfoto 2021-11-02 um 15 00 31" src="https://user-images.githubusercontent.com/47743312/139861911-5cd102ce-96f9-4761-8364-ccfcf1a48742.png">

I'll replace it back

Reply:
Oh right, I deleted more than I should have. Thanks